Makefile

```diff
@@ -24,13 +24,12 @@ runserver: build_docker  ## Run server inside XBlock Workbench container
 
 .PHONY: clean
 clean:  ## Remove build artifacts
-	deactivate || true
-	rm -rf venv/
+	coverage erase
```

Comment:
This is the proper way to cleanup the `reports/` directory,
not the `rm -rf reports/` below ;)